src/get_functions.jl

```diff
@@ -0,0 +1,19 @@
+
+"""
+get_origin(vtk_file)
+
+Retrieve the origin of a VTK regular structured grid file.
+
+"""
+
+function get_origin( vtk_file )
```

Comment:
```suggestion
"""
function get_origin(vtk_file)
```
Otherwise, the docstring is not attached to the function.

Reply:
I was not writing yet the docs but is good to know that, thanks!